envi/archs/arm/__init__.py

```diff
@@ -11,9 +11,11 @@
 class ArmModule(envi.ArchitectureModule):
 
     def __init__(self, name='armv6'):
+        import envi.archs.thumb16.disasm as eatd
```

Comment:
yes.  because arm and thumb are so interdependent, this has to happen somewhere.